spec/support/fixtures/assets/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can we delete this file?

Reply:
I added it because the integration tests depend on the `assets/` directory existing. If CI starts fresh each time, this directory won't exist and a [bunch of file/dir doesn't exist errors will be thrown](https://travis-ci.org/hanami/assets/builds/561441445?utm_source=github_status&utm_medium=notification). If you look at the build output for each commit here, you'll see what adding this file resolved.

CI wasn't running these tests; it was running tests in the non-existent `isolation` directory (so, no failures). I'm trying to re-incorporate the integration tests so the build is more reliable. As it is, there are several failing specs in master.

Reply:
I'll make a checklists of the remaining failing specs some time this weekend and ask for feedback 😄